docs/new-editor.md

```diff
@@ -101,12 +104,24 @@ client is expected to send a `workspace/didChangeConfiguration` notification
 containing user configuration right after the `initialized` notification.
 
 A full list of server configuration options can be found in
-[Configuration.scala][]. All configuration options have default values. An
+[Configuration.scala][]. An
 example of how the configuration options are used from the VS Code plugin can be
 seen in the
 [package.json](https://github.com/scalameta/metals/blob/master/vscode-extension/package.json)
 manifest.
 
+Here are the default values for all the options:
+
+```tut:passthrough
+{
+  import scala.meta.metals.Configuration
+  import io.circe.syntax._
+  println("```json")
+  println(Configuration().asJson.spaces2)
```

Comment:
tut:passthrough 🙏 it's so good

As a general recommendation, I think it's best to keep the code as much as possible in `*.scala` files and call from here. In this case we could implement this body in Configuration.scala to minimize the code in this markdown file. Alternatively, a `object Site { def defaults: String = ... }` that we could call with `Site.default`